lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Launcher.hs

```diff
@@ -150,8 +150,9 @@ spec = do
                     , "--genesis-block", secret
                     , "--secret", secret
                     ]
-            (Exit c, Stdout o, Stderr _) <- cardanoWalletCLI @t args
+            (Exit c, Stdout o, Stderr e) <- cardanoWalletCLI @t args
             c `shouldBe` ExitFailure 1
+            e `shouldBe` ""
             o `shouldContain` ("As far as I can tell, this isn't a valid block file: " <> secret)
```

Comment:
Suspicious: In this test we check for the message in stdout. The other tests check in stderr.

Reply:
CLI argument parsing failures are sent on stderr; then once the application has started, it sends its logs on stdout. In this case, this happens when the application has started though it should probably happens before.